pkg/service/credential/service.go

```diff
@@ -90,9 +88,44 @@ func NewCredentialService(config config.CredentialServiceConfig, s storage.Servi
 }
 
 func (s Service) CreateCredential(ctx context.Context, request CreateCredentialRequest) (*CreateCredentialResponse, error) {
-	returnFunc := s.createCredentialFunc(request)
+	watchKeys := make([]storage.WatchKey, 0)
+	var statusListCredential *StoredCredential
+
+	var slcMetadata StatusListCredentialMetadata
+
+	if request.isStatusValid() && (request.Suspendable || request.Revocable) {
```

Comment:
nit: may want to define a method such as `HasStatus` on the credential in place of `request.Suspendable || request.Revocable`